sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -187,54 +187,38 @@ public String getEntityPath() {
     }
 
     /**
-     * Abandon a {@link ServiceBusReceivedMessage message} with its lock token. This will make the message available
+     * Abandon a {@link ServiceBusReceivedMessage message}. This will make the message available
      * again for processing. Abandoning a message will increase the delivery count on the message.
      *
-     * @param lockToken Lock token of the message.
-     *
-     * @return A {@link Mono} that completes when the Service Bus abandon operation completes.
-     * @throws NullPointerException if {@code lockToken} is null.
-     * @throws UnsupportedOperationException if the receiver was opened in {@link ReceiveMode#RECEIVE_AND_DELETE}
-     *     mode.
-     * @throws IllegalArgumentException if {@code lockToken} is an empty value.
-     */
-    public Mono<Void> abandon(String lockToken) {
-        return abandon(lockToken, receiverOptions.getSessionId());
-    }
-
-    /**
-     * Abandon a {@link ServiceBusReceivedMessage message} with its lock token. This will make the message available
-     * again for processing. Abandoning a message will increase the delivery count on the message.
-     *
-     * @param lockToken Lock token of the message.
-     * @param sessionId Session id of the message to abandon. {@code null} if there is no session.
+     * @param message The {@link ServiceBusReceivedMessage} message.
      *
      * @return A {@link Mono} that completes when the Service Bus abandon operation completes.
      * @throws NullPointerException if {@code lockToken} is null.
      * @throws UnsupportedOperationException if the receiver was opened in {@link ReceiveMode#RECEIVE_AND_DELETE}
      *     mode.
      * @throws IllegalArgumentException if {@code lockToken} is an empty value.
      */
-    public Mono<Void> abandon(String lockToken, String sessionId) {
-        return abandon(lockToken, null, sessionId);
+    public Mono<Void> abandon(ServiceBusReceivedMessage message) {
+        return abandon(message, null);
     }
 
     /**
-     * Abandon a {@link ServiceBusReceivedMessage message} with its lock token and updates the message's properties.
+     * Abandon a {@link ServiceBusReceivedMessage message} updates the message's properties.
      * This will make the message available again for processing. Abandoning a message will increase the delivery count
      * on the message.
      *
-     * @param lockToken Lock token of the message.
+     * @param message The {@link ServiceBusReceivedMessage} message.
      * @param propertiesToModify Properties to modify on the message.
      *
      * @return A {@link Mono} that completes when the Service Bus operation finishes.
-     * @throws NullPointerException if {@code lockToken} is null.
+     * @throws NullPointerException if {@code lockToken} or {@code message} is null.
```

Comment:
There is no longer a lock token that's accepted. This probably applies to all the other docs.